### ISSUE TITLE
Placeholder and default properties in contenttype fields

### ISSUE THREAD

**Issue by Boorj** [bolt/bolt#4743]
Probably i'm wrong (using v2.3a6), but when i was looking into 
https://github.com/bolt/bolt/blob/release/2.2/app/view/twig/editcontent/fields/_text.twig 
https://github.com/bolt/bolt/blob/master/app/view/twig/editcontent/fields/_text.twig
and some other fields, i found that:
* Textarea is missing **placeholder** and needs:
```twig
placeholder: context.contenttype.fields[key].placeholder|default(''),
```
* lots of fields (integer, float, text, textarea, …) are missing the usage of **default** value. Here's the fix that i use:
```twig
{# for text #}
set value = context.content.get(contentkey)|default(context.contenttype.fields[key].default)|default('')
{# for numbers: float and integer #}
set value =  1 * context.content.get(contentkey)|default(context.contenttype.fields[key].default)|default(0)
```

**Comment by rarila** [comment]
As @CarsonF said on IRC these things should definitely be fixed/set in the php part.

**Comment by Boorj** [comment]
Yes, but since it's documented (for default values), it should be fixed somehow. So i just put that here to not forget.

**Comment by bobdenotter** [comment]
Fixed in #4771.

**Comment by Boorj** [comment]
Probably now we should remove my workarounds, since there's [this](https://github.com/rossriley/bolt/commit/66c18428982ca2e0b388493cc76d7167c7e62e11#diff-b60d5990f13034e72f21ef00bce39668R119) ?
